# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libwbxml2/no-cvs-clean.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

CVS directories are already removed by the ROCK build scripts.
-jsaw

--- wbxml2-0.8.2/Makefile.am~	2003-10-17 21:09:20.000000000 +0200
+++ wbxml2-0.8.2/Makefile.am	2004-01-10 18:18:43.000000000 +0100
@@ -24,7 +24,6 @@
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(wbxmldocdir)/manual
 	cp -Rp doc/* $(DESTDIR)$(wbxmldocdir)/manual
-	rm -r `find $(DESTDIR)$(wbxmldocdir)/manual -name "CVS"`
 
 dist-bz2: distdir
 	sed -e "s/tar.gz/tar.bz2/g" $(distdir)/wbxml2.spec > $(distdir)/wbxml2.spec.aux
